src/components/graph/nodes/network-modification-node.tsx

```diff
@@ -88,6 +89,12 @@ const NetworkModificationNode = (props: NodeProps<ModificationNode>) => {
     const studyUuid = useSelector((state: AppState) => state.studyUuid);
     const currentRootNetworkUuid = useSelector((state: AppState) => state.currentRootNetworkUuid);
 
+    const intl = useIntl();
+    const [tooltipOpen, setTooltipOpen] = useState(false);
```

Comment:
Add a comment to explain why we used this state to manage the tooltip, i'm worried someone would try to simplify this code by removing this behaviour